content/lang/collections/traitattr/occurattributes.es.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/*
+------------------
+Language: Espanol
+Translated by: Google Translation
+Date Translated: 01/31/2024
+------------------
+*/
+
+$LANG['OCC_ATTRIBUTE_BATCH_EDIT'] =  'Editor por lotes de atributos de ocurrencia';
+$LANG['Home'] = 'Inicio';
```

Comment:
Needs caps (check this throughout this file)
Also make sure that all changes made to variable names in occurattributes.en.php are reflected in the .es and .fr translations